internal/dev_server/dev_server.go

```diff
@@ -67,3 +71,15 @@ func RequestErrorHandler(w http.ResponseWriter, r *http.Request, err error) {
 	log.Printf("Error while serving request: %+v", err)
 	http.Error(w, err.Error(), http.StatusBadRequest)
 }
+
+func getDBPath() string {
+	statePath := os.Getenv("XDG_STATE_HOME")
+	if statePath == "" {
+		home, _ := homedir.Dir()
+		statePath = filepath.Join(home, ".local/state")
+	}
```

Comment:
By the way, just realised. This translates to `~/.local/state/dev_server.db`, we probably want `~/.local/state/ldcli/dev_server.db` or something.

Worth using [`xdg.StateFile("ldcli/dev_server.db")`](https://github.com/adrg/xdg) instead.

Reply:
Oh! Nice. Yeah. I'll do this instead.